sklearn/cluster/optics_.py

```diff
@@ -219,8 +221,10 @@ def fit(self, X, y=None):
 
         Parameters
         ----------
-        X : array, shape (n_samples, n_features)
-            The data.
+        X : array, shape (n_samples, n_features), or (n_samples, n_samples)  \
+if metric=’precomputed’.
```

Comment:
I prefer this indented... I don't see any reason to have it flush left.
```suggestion
                if metric=’precomputed’.
```

Reply:
See #5078, we do similar things in some latest classes, e.g., ColumnTransformer

Reply:
If I had time to pull this into an ancient discussion of this at numpydoc, I would... Thanks